### ISSUE TITLE
Training pattern differes with different number of worker when using multiprocessing

### ISSUE THREAD

**Issue by tariksetia** [plaidml/plaidml#374]
I am using Keras with plaidml as backend on MacBook Pro 2018.
When I train with `use_mutliprocessing=True` and default workers. The learning plots seems be ok. However when I increase ### the no of workers to 10, training starts with very high loss value and train loss doesn't converge much. 
e.g with single worker, training loss at start is in range of [10,15], but when using multiprocessing with more than on worker, loss at start is in range of [56, 115]
Is this normal? or is this an Issue?

<img width="309" alt="image" src="https://user-images.githubusercontent.com/1137658/61632987-c2da9200-acab-11e9-81a0-cc97384ce2de.png">

blue is the curve with single worker and orange curve was obtained while training with 10 workers.

**Comment by tzerrell** [comment]
Probably the easiest way to test if this is an issue is to try using a different backend (e.g. tensorflow). This might be substantially slower, but the pattern shows up quite early so hopefully that won't be too much of an issue. If the failure to converge replicates with another backend, then this is expected behavior. If not, let us know that this is a PlaidML-specific issue.

If this does appear to be PlaidML-specific, having enough details to replicate the behavior on our end will make this much easier to debug. (I.e., model, training data, hyperparameters, ...)
